InternetShop/InternetShop/User.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+namespace InternetShop
+{
+    public class User
+    {
+        public int? actor_id { get; set; }
+        public string first_name { get; set; }
+        public string last_name { get; set; }
```

Comment:
Usually we write property name with capital letter

Reply:
done

Reply:
done